### ISSUE TITLE
Can't install laravel 9 with codeception due to nanomaduro/collision

### ISSUE THREAD

**Issue by jvrsolis** [Codeception/Codeception#6362]
#### What are you trying to achieve?
Installing Laravel 9. 

#### What do you get instead?
Unable to install laravel 9. **Due to nanomaduro/collision upgrade that requires symfony/console v6.1.0 and the latest codeception v4.1.29 has an upper bound of <6.0 for symfony/console**
> Provide console output if related. Use `-vvv` mode for more details.

```bash
 - Conclusion: don't install codeception/codeception 4.1.29 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.1 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.2 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.1.0 (conflict analysis result)
    - Conclusion: don't install symfony/console v6.0.0-BETA1 (conflict analysis result)
    - Conclusion: don't install symfony/console v6.0.0-RC1 (conflict analysis result)
```
> Provide test source code if related

```php
// paste test
```
### Details

* Codeception version: 4.1.29
* PHP Version: 8.0.3
* Operating System: linux
* Installation type: Phar || Composer
* List of installed packages (`composer show`)
* Suite configuration:

```yml
# paste suite config here
```

**Comment by W0rma** [comment]
Support for symfony 6 will be added in the next major version 5.0 (https://github.com/Codeception/Codeception/blob/5.0/composer.json)

**Comment by jvrsolis** [comment]
You mean 6.1 right? symfony 6 is already supported in 4.1.29

**Comment by Naktibalda** [comment]
4.1 really doesn't support symfony 6.

**Comment by Naktibalda** [comment]
You are welcome to try to use Codeception 5:
```
"codeception/codeception": "^5.0.0-alpha2",
"codeception/module-laravel5": "dev-codecept5",
```

Please let me know if you have any issues.

**Comment by codisfy** [comment]
> You are welcome to try to use Codeception 5:
> 
> ```
> "codeception/codeception": "^5.0.0-alpha2",
> "codeception/module-laravel5": "dev-codecept5",
> ```
> 
> Please let me know if you have any issues.

you meant 
```
"codeception/codeception": "^5.0.0-alpha2",
"codeception/module-laravel": "dev-codecept5",
```

**Comment by Naktibalda** [comment]
I released `codeception/module-laravel` 3.0.0 with support for Codeception 5, but it only supports Laravel 8.
@TavoNiievez wrote that Laravel 9 support needs more work.
